new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -679,16 +685,23 @@ public static boolean createPvc(V1PersistentVolumeClaim persistentVolumeClaim) t
       throw new IllegalArgumentException(
           "'namespace' field in the metadata cannot be null when calling createPvc()");
     }
-
+    logger.info("creating persistent volume claim" + persistentVolumeClaim.getMetadata().getName());
     String namespace = persistentVolumeClaim.getMetadata().getNamespace();
 
-    V1PersistentVolumeClaim pvc = coreV1Api.createNamespacedPersistentVolumeClaim(
-        namespace, // name of the Namespace
-        persistentVolumeClaim, // persistent volume claim configuration data
-        PRETTY, // pretty print output
-        null, // indicates that modifications should not be persisted
-        null // fieldManager is a name associated with the actor
-    );
+    try {
+      KubernetesApiResponse<V1PersistentVolumeClaim> response = pvcClient.delete(namespace,
```

Comment:
A delete call doesn't belong in a create method

Reply:
Its added by @lennyphan , Lenny would you like to add you comments ?

Reply:
This still needs to be addressed.

Reply:
removed the delete call.